.detekt.yml

```diff
@@ -70,6 +70,11 @@ style:
     maxDestructuringEntries: 4
   ForbiddenComment:
     active: false
+  ForbiddenMethodCall:
+    active: true
+    # Use org.ossreviewtoolkit.utils.ort.runBlocking instead in all code that can be used as a library.
```

Comment:
IMO this line can be dropped in favor of keeping the per-entry comment in line 77.

Reply:
I wanted to provide a bit more detail here, I thought someone might first look for an explanation here before searching the Git history. But I forgot to remove the comment in line 77.